ksiren/src/main/java/com/brightspace/ksiren/Action.kt

```diff
@@ -41,24 +63,24 @@ data class Action(
 					"class" -> {
 						reader.beginArray()
 						while (reader.hasNext()) {
-							conditionalRead(reader, {classes.add(it)})
+							conditionalRead(reader, { classes.add(it) })
```

Comment:
nit: IDE warning re: Kotlin style:

![image](https://user-images.githubusercontent.com/2973055/90289797-a5461080-de4a-11ea-808f-a54aef79c56b.png)

Reply:
Yeah, those lines show up in the diff because I did a "Reformat Code" operation to ensure the net-new code was formatted according to EditorConfig rules, but the existing code didn't either - since it was simple whitespace changes, I decided to keep them in since it can be filtered from the PR easily, and I was planning on doing a whole-project lint/warning fixup PR after this just to get things back in order. Mind if I defer this until then?